tests/network/dry_run/test_dry_run_kubemacpool.py

```diff
@@ -130,11 +117,12 @@ def dry_run_vm_with_mac_address(
 def test_dry_run_mac_not_saved(
     dry_run_vma_mac_address,
     vm_with_mac_address,
-    running_vm_mac_addresses,
 ):
     """Assure that the MAC address given to a VM created in dry-run mode is still available in the MAC-pool."""
-    assert dry_run_vma_mac_address == running_vm_mac_addresses, (
-        f"{vm_with_mac_address.name} MAC address is {running_vm_mac_addresses}"
+    vm_interfaces = vm_with_mac_address.get_interfaces()
```

Comment:
I don't think lines 122+123 are part of the test. They are what the test require as part of it - part of the test's setup - and therefor should be in a fixture.

Reply:
Reading the interfaces is not part of the test?
We are back to claiming that the test body is just the assertion, which I strongly disagree.

In this case, even the creation of the VM as dry-run should be part if the test, but it is not in the scope of this change.

Reply:
It was in a fixture before your change, so it make sense it'll stay, there. But as we discussed before - both approaches have their pros and cons, you can continue with this direction, I won't force mine. You can mark this thread as resolved.

Reply:
OK, thanks.